<launch>

  <arg name="rviz" default="true" />

  <node name="ximea" pkg="ximea_camera" type="ximea_camera_node" output="screen" >
    <param name="frame_rate" type="int" value="20" />
    <param name="bandwidth_mbps" type="int" value="2000" />
    <rosparam param="camera_param_file_paths" subst_value="true">[$(find ximea_camera)/config/custom_ximea.yaml]</rosparam>
  </node>

  <group if="$(arg rviz)">
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ximea_camera)/config/vis.rviz" />
  </group>

</launch>
